RELEASE.md

```diff
@@ -115,15 +115,37 @@ The Release Manager must create a release candidate branch:
 
     -   [ ] 8.1.  Cherry-pick fixes
 
-        At this stage, it is possible for the build to have failed. The fixes for this failure should have been merged to `master` following a successfully approved Pull Request.
-
-        You can cherry-pick the required changes with:
-
-        ```bash
-        git cherry-pick <commit-hash>
-        ```
-
-        Ensure the cherry-pick commit is free from any conflicts.
+        - [ ] 8.1.1. Fixes on codacy-chart
+        
+            At this stage, it is possible for the build to have failed. The fixes for this failure should have been merged to `master` following a successfully approved Pull Request.
+
+            You can cherry-pick the required changes with:
+
+            ```bash
+            git cherry-pick <commit-hash>
+            ```
+
+            Ensure the cherry-pick commit is free from any conflicts.
```

Comment:
This can all go up one level:

```suggestion
    - [ ] 8.1. Cherry-picking fixes on codacy-chart
        
        At this stage, it is possible for the build to have failed. The fixes for this failure should have been merged to `master` following a successfully approved Pull Request.

        You can cherry-pick the required changes with:

        ```bash
        git cherry-pick <commit-hash>
        ```

        Ensure the cherry-pick commit is free from any conflicts.
```